MvvmCross-iOSSupport/MvvmCross.iOS.Support.XamarinSidebar/MvxSidebarPresenter.cs

```diff
@@ -33,94 +32,77 @@ protected virtual bool PresentationHintHandler(MvxPanelPresentationHint hint)
             return true;
         }
 
-        public override void Show(MvxViewModelRequest request)
+        protected override void RegisterAttributeTypes()
         {
-            IMvxIosView viewController = Mvx.Resolve<IMvxIosViewCreator>().CreateView(request);
-            Show(viewController);
+            base.RegisterAttributeTypes();
+
+            _attributeTypesToShowMethodDictionary.Add(
+                typeof(MvxSidebarPresentationAttribute),
+                (vc, attribute, request) => ShowSidebarViewController(vc, attribute as MvxSidebarPresentationAttribute, request));
```

Comment:
Casting the attribute using the `as` keyword could result in a `null` value (meanly if the attribute is not of type `MvxSidebarPresentationAttribute` (which theoretically is possible). 

So either make a direct cast (which is better performance wise) and in case of wrong usage it will generate an exception on the correct place. Or if we don't want the application to crash we should do a check.